deprecate stackdf and meltdf (#2031)
JuliaData/DataFrames.jl: docs/src/man/reshaping_and_pivoting.md

│ 6   │ SepalWidth │ 3.0      │ virginica     │
```

> If you prefer prefers to specify the id columns then use `Not` with `stack` like this:

Reviewer: ```suggestion
If you prefer to specify the id columns then use `Not` with `stack` like this:
```

│ 6   │ setosa        │ 6     │ 1.7         │ 0.4        │ 5.4         │ 3.9        │
```

> If you pass `view=true` to `stack` then you get a data frame whose columns are views into the original wide data frame. Here is an example:

Reviewer: Maybe use a more impersonal formulation?
```suggestion
Passing `view=true` to `stack` returns a data frame whose columns are views into the original wide data frame. Here is an example:
```
